lib/screens/speaker/speaker.dart

```diff
@@ -36,17 +43,90 @@ class SpeakerScreen extends StatelessWidget {
   }
 
   Widget _buildSuccess(BuildContext context, List<Speaker> speakerList) {
-    //TODO: UI
-    return Center(
-      child: Column(
-        mainAxisSize: MainAxisSize.min,
-        children: <Widget>[
-          Text(speakerList[0].name, textAlign: TextAlign.center),
-        ],
+    double height = MediaQuery.of(context).size.height;
+    double heightFactor = height / 1000;
+    double bottom = MediaQuery.of(context).viewInsets.bottom;
+
+    List<Widget> sL = [];
+    speakerList.forEach((element) {
+      sL.add(_cardsLoader(element));
+    });
```

Comment:
Attempt to use labda functions wherever you can.
```suggestion
    speakerList.forEach((e) => sL.add(_cardsLoader(e)););
```